lib/Client/HangarAPIClient.php

```diff
@@ -492,11 +477,13 @@ public function getProjectsPinnedByUser(string $username): array
      * Get a list of hangar staff
      *
      * Requires the view_public_info permission
+     * @param string $query Search query
      * @param RequestPagination|null $pagination
+     * @param string|null $sort Optional name of the field to sort the results by
      * @return StaffList
      * @throws ApiException
      */
-    public function getStaff(?RequestPagination $pagination = null): StaffList
+    public function getStaff(string $query, ?RequestPagination $pagination = null, ?string $sort = null): StaffList
```

Comment:
You could probably default initialize query as "".
It would also be better to have an enum for the sort parameter